prosody/Dockerfile

```diff
@@ -2,7 +2,13 @@ FROM jitsi/base
 
 RUN \
 	apt-dpkg-wrap apt-get update && \
-	apt-dpkg-wrap apt-get install -t stretch-backports -y prosody && \
+	apt-dpkg-wrap apt-get install -t stretch-backports -y prosody \
+		sasl2-bin libsasl2-modules-ldap lua-ldap \
+		luarocks git libsasl2-dev gcc liblua5.2-dev && \
+	git clone https://github.com/JorjBauer/lua-cyrussasl.git && \
```

Comment:
Can't we use the already packaged version? https://packages.debian.org/stretch/lua-cyrussasl

Reply:
No, because it compiled for lua5.1 (https://packages.debian.org/stretch/amd64/lua-cyrussasl/filelist), but we using lua5.2

Reply:
Damn.